tensorstore/driver/ometiff/BUILD

```diff
@@ -0,0 +1,134 @@
+# OMETiff TensorStore driver
+
+load("//tensorstore:tensorstore.bzl", "tensorstore_cc_library", "tensorstore_cc_test")
+load("//docs:doctest.bzl", "doctest_test")
+
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+DOCTEST_SOURCES = glob([
+    "**/*.rst",
+    "**/*.yml",
+])
+
+doctest_test(
+    name = "doctest_test",
+    srcs = DOCTEST_SOURCES,
+)
+
+filegroup(
+    name = "doc_sources",
+    srcs = DOCTEST_SOURCES,
+)
+
+tensorstore_cc_library(
+    name = "ometiff",
+    deps = [
+            ":driver",
+    ],
+)
+
+tensorstore_cc_library(
+    name = "metadata",
+    srcs = ["metadata.cc"],
+    hdrs = ["metadata.h"],
+    deps = [
+        "//tensorstore:array",
+        "//tensorstore:codec_spec",
+        "//tensorstore:data_type",
+        "//tensorstore:schema",
+        "//tensorstore:strided_layout",
+        "//tensorstore/index_space:index_transform",
+        "//tensorstore/internal:data_type_endian_conversion",
+        "//tensorstore/internal:flat_cord_builder",
+        "//tensorstore/internal:json_metadata_matching",
+        "//tensorstore/internal:type_traits",
+        "//tensorstore/internal/json_binding",
+        "//tensorstore/internal/json_binding:bindable",
+        "//tensorstore/internal/json_binding:data_type",
+        "//tensorstore/internal/json_binding:dimension_indexed",
+        "//tensorstore/serialization",
+        "//tensorstore/serialization:json",
+        "//tensorstore/util:result",
+        "//tensorstore/util:span",
+        "//tensorstore/util/garbage_collection",
+        "@com_github_nlohmann_json//:nlohmann_json",
+        "@com_google_absl//absl/algorithm:container",
+        "@com_google_absl//absl/base:endian",
+        "@com_google_absl//absl/status",
+        "@com_google_absl//absl/strings",
+    ],
+    alwayslink = 1,
+    linkopts = ["/usr/lib/x86_64-linux-gnu//libtiffxx.so.5",
```

Comment:
This assumes a system-installed libtiff in a particular location, which we can't do for tensorstore --- instead, just add `@libtiff//:tiff` to `deps` since we already have libtiff available as a dependency.

Reply:
My bad, I actually don't need it here anymore.